src/js/firstparties/google.js

```diff
@@ -0,0 +1,26 @@
+/* globals findInAllFrames:false */
+(function() {
```

Comment:
We don't need to wrap our content scripts in self-executing functions (since content scripts operate in their own JavaScript context), right?

Reply:
Ah, didn't realize they had their own contexts. Thanks for the heads up.